download-notebooks.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Notebook directory used by JupyterHub in birdhouse-deploy
+NOTEBOOK_DIR="/notebook_dir/tutorial-notebooks"
+
+# Download notebooks required for the base image and add them to the notebook directory
+wget -O - https://github.com/Ouranosinc/pavics-sdi/archive/master.tar.gz | \
+    tar -xz --wildcards -C $NOTEBOOK_DIR --strip=4 "*/docs/source/notebooks/jupyter_extensions.ipynb"
```

Comment:
The biggest missing feature of my previous implemention is no pluggable/customizable list of notebooks so each org deployment of PAVICS can choose what to deploy.  A default list should be include for bootstrapping of course.

So I do not see this pluggable feature here.  Or maybe you can add more docs explaining how it could be done?  Each org override the `download-notebooks.sh` with their own implementation?